docs/components/Drawer/Web.stories.tsx

```diff
@@ -11,6 +11,7 @@ export default {
   component: Drawer,
   parameters: {
     viewMode: "story",
+    previewTabs: { code: { hidden: false } },
```

Comment:
need to overwrite the imports for this

<img width="1305" alt="image" src="https://github.com/GetJobber/atlantis/assets/15986172/929fc7a4-99b3-41dd-b483-2175a70ac940">

Reply:
Ooops! That turned into a waterfall of missing imports 😓  Code tab is working now.

Reply:
Wait, I know Michael is changing things. Disregard the re-request for review for now!

Reply:
Boom. Done.